### DISCUSSION TITLE
Multi Monitor Setup

### DISCUSSION THREAD

**Discussion by 4thehalibit** [prasanthrangan/hyprdots#782] [Q&A]
I was setting up my multi monitors and in doing so I am now unable to change my wallpaper. I have set the monitors using  _user prefs_  I have also install _nwg-displays_. 
`monitor=DP-3, 1920x1080, 0x0 1 `
`monitor=DP-2, 1440x900, 1920x0 1
`

**Comment by kRHYME7** [comment]
what happens if you run ``` ~/.config/hypr/scripts/swwwallpaper.sh ``` ?

**Reply by 4thehalibit** [reply]
Heads up I hope i ran this correctly. Only been using arch for a few weeks and linux for a year

[vwestberg@archy ~]$ sudo ~/.config/hypr/scripts/swwwallpaper.sh
[sudo] password for vwestberg: 
jq: parse error: Invalid numeric literal at line 1, column 28
jq: parse error: Invalid numeric literal at line 1, column 28
grep: /root/.config/hypr/theme.ctl: No such file or directory
ERROR: wallpaper  not found...
[vwestberg@archy ~]$ `

**Reply by kRHYME7** [reply]
> Heads up I hope i ran this correctly. Only been using arch for a few weeks and linux for a year

>[vwestberg@archy ~]$ sudo ~/.config/hypr/scripts/swwwallpaper.sh
[sudo] password for vwestberg:
jq: parse error: Invalid numeric literal at line 1, column 28
jq: parse error: Invalid numeric literal at line 1, column 28
grep: /root/.config/hypr/theme.ctl: No such file or directory
ERROR: wallpaper not found...
[vwestberg@archy ~]$ `

> linux for a year

this seems pretty alarming why sudo the script ?

**Comment by 4thehalibit** [comment]
I am marking this as closed. I am pretty sure this has o do with my hardware.
